packages/react-app/src/views/CreateZone.jsx

```diff
@@ -24,11 +24,10 @@ function Polygons({ writeContracts, address, injectedProvider, readContracts, us
     pitch: 0,
     // padding: { top: 0, bottom: 0, left: 0, right: 0 },
   });
-  const [message, setMessage] = useState(null);
   const [map, setMap] = useState(null);
 
   // custom hooks
-  useFlyTo(map, setViewState);
+  useFlyTo(map, setViewState, 10);
```

Comment:
So this means we zoom in a bit when we create a new zone? Since the default zoom is 18.

Reply:
Yes, the zone creation route has more zoom because it was too far away for any practical use.